src/display/annotation_layer.js

```diff
@@ -3919,6 +3975,34 @@ class AnnotationLayer {
     return this.#editableAnnotations.get(id);
   }
 
+  addFakeAnnotation(editor) {
```

Comment:
It would be good to have a docstring or other documentation that explains the purpose of the fake annotation (i.e., what do we need it for?).